app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,58 @@
+class ArticlesController < ApplicationController
+  before_action :authenticate_user!, except: [:index]
```

Comment:
This line is unnecessary as petergate will handle all of that.

Reply:
sweet. i'll remove that